### ISSUE TITLE
Deferred deeplink attribution issue

### ISSUE THREAD

**Issue by MaxenceLeroy** [adjust/ios_sdk#594]
Hello. 

I have setup Adjust in my projet. What is working:
- Events tracking
- Open deeplink
- Create a deeplink that redirects to the store

But the attribution is always set to "organic" when I use a deferred deeplink (before doing it, I always forget the device in the adjust testing console).

I have tested the following situation:
- I install the app with my deeplink
- I do not open it
- I click again on the link
- The attribution is set to the right tracker

I did not implement the Adjust Delegate, so according to the documentation "the Adjust SDK will always try to open the deep link by default.".

Important things to note, when I am trying in dev environment to open the link, then open the app with Xcode. In this case the `adjustDeeplinkResponse` function from the callback is never called, but the `application(_ app: UIApplication, open url: URL)` is called with the following URL: "myappscheme://google/link/?dismiss=1&is_weak_match=1" (I have firebase installed on my project)

Is there anything I can do to fix this problem ?

Thanks in advance and excuse me if there is something missing in this issue.

**Comment by uerceg** [comment]
Hi @MaxenceLeroy 

Thank you for detailed explanation.

Can you share the tracker URL which you are using to test (the one with the deep link which you're expecting to be delivered to your app after installing and opening it)?

In the meantime, a thing for you to check: In dashboard when you find `your tracker in list of trackers -> click gear icon next to it -> EDIT -> Attribution Settings -> CLICK BASED ATTRIBUTION -> Probabilistic Matching` option. Is it turned ON or OFF for tracker you are using for testing? If it's OFF, turn it ON and give it a shot.

To answer your comment about `adjustDeeplinkResponse:` callback not being triggered when you open your app with click on deep link -> that's expected. That callback method is meant to be triggered by SDK only once SDK receives deferred deep link when install is tracked (pretty much in scenario which you're now trying to see in action). After we solve this testing setup issues, you should also see this callback being triggered once deferred deep link from tracker URL gets delivered into your app.

In case you have any further questions, feel free to ping.
